### ISSUE TITLE
{Linked:Bug108199} [Adaptive Cards > Designer] - Ensures <img> elements have alternate text or a role of none or presentation 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8700]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/explorer/Action.ToggleVisibility.html
OS Version: 23H3(OS Build 25951.1010)
Browser Version: Edge Dev 117.0.2033.0
Tool: Accessibility Insight For Web

**Pre-Requisite:**
Run the below attached JSON file under 'Copy Card Payload editor.'

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Run the Json file.
3. Navigate to the Preview mode and press enter key to activate it.
4. Navigate the screen and observe the issue.

**Actual Result:**
<img> elements does not have alternate text or a role of none or presentation.

**Expected Result:**
<img> elements should have alternate text or a role of none or presentation.  

**Element path:**
 .ac-pushButton.style-default.ac-selectable:nth-child(5) > img

**Snippet:**
<img src="https://res.cdn.office.net/edureflect/public/20230921.6/feedbackEmojis/VeryHappy.png" style="width: 16px; height: 16px; flex: 0 0 auto; max-height: 100%;">

**How to fix:** 
Fix any of the following:

1. Element does not have an alt attribute
2. aria-label attribute does not exist or is empty
3. aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
4. Element has no title attribute
5. Element's default semantics were not overridden with role="none" or role="presentation"

**User Impact:**
Screen reader users will not able to know name of emotion icons if alt attribute is not defined.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

![WCAG1 1 1-Ensures img elements have alternate text or a role of none or presentation](https://github.com/microsoft/AdaptiveCards/assets/95913748/f7d0bf90-acd8-4861-bf8b-e9aacc856778)

![WCAG1 1 1-Ensures img elements have alternate text or a role of none or presentations](https://github.com/microsoft/AdaptiveCards/assets/95913748/a44641f5-ec66-402b-b981-3c0ae2391d1d)

### Card JSON

```JSON
{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "TextBlock",
            "size": "Medium",
            "weight": "Bolder",
            "text": "${title}"
        },
        {
            "type": "ColumnSet",
            "columns": [
                {
                    "type": "Column",
                    "items": [
                        {
                            "type": "Image",
                            "style": "Person",
                            "url": "${creator.profileImage}",
                            "size": "Small"
                        }
                    ],
                    "width": "auto"
                },
                {
                    "type": "Column",
                    "items": [
                        {
                            "type": "TextBlock",
                            "weight": "Bolder",
                            "text": "${creator.name}",
                            "wrap": true
                        },
                        {
                            "type": "TextBlock",
                            "spacing": "None",
                            "text": "Created {{DATE(${createdUtc},SHORT)}}",
                            "isSubtle": true,
                            "wrap": true
                        }
                    ],
                    "width": "stretch"
                }
            ]
        },
        {
            "type": "TextBlock",
            "text": "${description}",
            "wrap": true
        },
        {
            "type": "FactSet",
            "facts": [
                {
                    "$data": "${properties}",
                    "title": "${key}:",
                    "value": "${value}"
                }
            ]
        }
    ],
    "actions": [
        {
            "type": "Action.ShowCard",
            "title": "Set due date",
            "card": {
                "type": "AdaptiveCard",
                "body": [
                    {
                        "type": "Input.Date",
                        "id": "dueDate"
                    },
                    {
                        "type": "Input.Text",
                        "id": "comment",
                        "placeholder": "Add a comment",
                        "isMultiline": true
                    }
                ],
                "actions": [
                    {
                        "type": "Action.Submit",
                        "title": "OK"
                    }
                ],
                "$schema": "http://adaptivecards.io/schemas/adaptive-card.json"
            }
        },
        {
            "type": "Action.OpenUrl",
            "title": "View",
            "url": "${viewUrl}"
        },
        {
            "type": "Action.OpenUrl",
            "title": "",
            "iconUrl": "https://res.cdn.office.net/edureflect/public/20230921.6/feedbackEmojis/VeryHappy.png",
            "url": "https://eduta-reflect-int.trafficmanager.net/reflectTeamAppIndex.html?tid=ff206b96-1c1d-4baf-b50a-d011a885b18c&userObjectId=7deb1cb5-6df8-4ccb-9e03-033b00d103ab&initLocale=en-us&initTheme=default&entityId=ReflectEntityIdInt&hostClientType=web"
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.5"
}
```

**Comment by anna-dingler** [comment]
@jwoo-msft I think this is a schema change?

**Comment by jwoo-msft** [comment]
> @jwoo-msft I think this is a schema change?
> 
> Or we need to apply tooltip to the image's altText?
@anna-dingler ,  using tooltip to describe button (which should be already done) & setting the role of image to presentation seems to be sufficient?

**Comment by anna-dingler** [comment]
> > @jwoo-msft I think this is a schema change?
> > Or we need to apply tooltip to the image's altText?
> 
> @anna-dingler , using tooltip to describe button (which should be already done) & setting the role of image to presentation seems to be sufficient?

Good call! Interestingly, we set role="presentation" for inline actions, but not regular ones
